### DISCUSSION TITLE
Shortcuts icon colors aren't the same

### DISCUSSION THREAD

**Discussion by NikunjK8** [fast4x/RiMusic#1877] [General]
I'm not sure if this is bug or happening to me only so I'm opening a discussion for time being. The app shortcuts have different colors. Can you confirm?

![IMG_20240515_094622](https://github.com/fast4x/RiMusic/assets/99175083/fbd96d24-1af4-4133-85e8-4637988f6d42)

**Comment by fast4x** [comment]
I'll check it
